src/Sulu/Bundle/AdminBundle/Resources/js/components/Backdrop/tests/Backdrop.test.js

```diff
@@ -8,12 +9,12 @@ test('The component should render', () => {
     expect(container).toMatchSnapshot();
 });
 
-test('The component should call a function when clicked', () => {
+test('The component should call a function when clicked', async() => {
     const onClickSpy = jest.fn();
     render(<Backdrop onClick={onClickSpy} />);
     const backdrop = screen.queryByTestId('backdrop');
 
     expect(onClickSpy).toHaveBeenCalledTimes(0);
-    fireEvent.click(backdrop);
+    await userEvent.click(backdrop);
```

Comment:
the `@testing-library/user-event` package will simulate user interactions morea realistically than the `fireEvent` method. the `fireEvent` method will only fire a single `click` event while the `@testing-library/user-event` will trigger a `hover` and a `mousedown` event before firing the `click` event like a real user would do.